docker/compose/aptos-node/haproxy.cfg

```diff
@@ -178,3 +162,12 @@ backend validator-api
     mode http
     default-server maxconn 1024
     server validator validator:8080
+
+frontend stats
+    mode http
+    bind :9102
+    option http-use-htx
+    http-request use-service prometheus-exporter if { path /metrics }
+    stats enable
+    stats uri /stats
```

Comment:
did we test this is actually accessible via kubectl ?

Reply:
As long as the haproxy metrics port is configured in the docker-compose file its accessible just like the rest.